README.md

```diff
@@ -149,3 +187,35 @@ so BT will know where the block is, and it can use the offset to seek in the dis
     }
 }
 ```
+
+
+## Limitation
+
+
+- Making a torrent cost lots of time due to sha-1 hash need to be done on every single piece of data.
+
+- EZIO will be extremely slow when the number of clients is too small.
+
+- Due to partclone limitation, for unsupported filesystem, sector-to-sector copy is done by dd in EZIO.
+
+## Future
+
+- Hope to become a sub-module in Clonezilla
+- Make a entire disk, including partition, a torrent
+
```

Comment:
add this to this section
`we are waiting for BEP-0039 to speed up the whole process`